README.md

```diff
@@ -17,21 +21,175 @@ python --version
+
+One can see all of the available options by:
+
+```bash
+    $: make
+
+    Available rules:
+
+  api-build                 Build API image
+  api-start                 Start API image container
+  api-stop                  Stop API image container
+  api-web                   Open API in web browser
+  clean                     Removes artifacts from the build stage, and other common Python artifacts.
+  clean-build               Remove build artifacts
+  clean-pyc                 Removes Python file artifacts
+  clean-secrets             Removes secret artifacts - Serverless
+  clean-test                Remove test and coverage artifacts
+  create-environment        Creates the Python environment
+  create-envrc              Set up the envrc file for the project.
+  delete-environment        Deletes the Python environment
+  delete-envrc              Delete the local envrc file of the project
+  destroy                   Remove ALL of the artifacts + Python environments
+  docker-local-dev-build    Build local development image
+  docker-local-dev-login    Start a shell session into the docker container
+  docker-local-dev-start    Start service for local development
+  docker-local-dev-stop     Stop service for local development
+  docker-prune              Clean Docker images
+  git-flow-install          Install git-flow
+  init                      Initialize the repository for code development
+  lint                      Run the 'pre-commit' linting step manually
+  pip-upgrade               Upgrade the version of the 'pip' package
+  pre-commit-install        Installing the pre-commit Git hook
+  pre-commit-uninstall      Uninstall the pre-commit Git hook
+  requirements              Install Python dependencies into the Python environment
+  show-params               Show the set of input parameters
+  sort-requirements         Sort the project packages requirements file
+```
+
+> **NOTE**: If you're using `Windows`, you may have to copy and modify to some
+> extents the commands that are part of the `Makefile` for some tasks.
+
+### Starting up the Docker container and initializing the repository
+
+In order to work on current / new features, one can use *Docker* to
+start a new container and start the local development process.
+
+To build the Docker image, one must follow the following steps:
+
+1. Start the Docker daemon. If you're using Mac, one can use the
+Docker Desktop App.
+2. Go the project's directory and run the following command using the `Makefile`:
+```bash
+# Go the project's directory
+cd /path/to/directory
+
+# Build the Docker iamge and start a container
+make docker-local-dev-start
+```
+3. Log into the container
+```bash
+# Log into the container
+make docker-local-dev-login
+```
+
+4. Once you're inside the container, you'll see the following prompt:
+
+```bash
+# Log into the container
+➜$:  ai-image-analyzer git:(feature/6-adding-docker-and-local-development) make docker-local-dev-login
```

Comment:
For the example branch, I think we could use `develop` rather than `feature/6-adding-docker-and-local-development`. How's that sound?

Reply:
Yes, definitely. I fixed this in the next set of updates. Thanks!